### PR TITLE
-collections add all files

### PR DESCRIPTION
**Pull request by Nikitav77**
add all file

### CODE REVIEW

File: src/com/nkita/practice/collections/PersonStatisticHandler.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        List<Person> personList = new ArrayList<>();
+
+        for (Person person : persons) {
+            if (nameList.contains(person.getName())) {
+                personList.add(person);
+
+            }
+        }
+
+//        if (personList.isEmpty()) {
```
Comment (Ilya1222) [inline src/com/nkita/practice/collections/PersonStatisticHandler.java]: Remove unnecessary comment

File: src/com/nkita/practice/collections/PersonStatisticHandler.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+
+    public void matchByNameIgnoreCase(String string) {
+        List<Person> personList = new ArrayList<>();
+        for (Person person : persons) {
+            if (person.getName().toLowerCase().contains(string.toLowerCase())) {
+                personList.add(person);
+            }
+        }
+//        if (!personList.isEmpty()) {
```
Comment (Ilya1222) [inline src/com/nkita/practice/collections/PersonStatisticHandler.java]: Remove unnecessary comment

File: src/com/nkita/practice/collections/PersonStatisticHandler.java [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        for (Person person : persons) {
+            personSet.add(person);
+            System.out.println(personSet);
+            return;
+        }
+
+    }
+
+
+//    public void removeDuplicates() {
```
Comment (Ilya1222) [inline src/com/nkita/practice/collections/PersonStatisticHandler.java]: Remove unnecessary comment

File: src/com/nkita/practice/collections/PersonStatisticHandler.java (hunk 1 above)
Comment (Nikitav77) [reply src/com/nkita/practice/collections/PersonStatisticHandler.java]: done

File: src/com/nkita/practice/collections/PersonStatisticHandler.java (hunk 2 above)
Comment (Nikitav77) [reply src/com/nkita/practice/collections/PersonStatisticHandler.java]: done

File: src/com/nkita/practice/collections/PersonStatisticHandler.java (hunk 3 above)
Comment (Nikitav77) [reply src/com/nkita/practice/collections/PersonStatisticHandler.java]: done

File: src/com/nkita/practice/collections/PersonStatisticHandler.java (hunk 1 above)
Comment (Ilya1222) [reply src/com/nkita/practice/collections/PersonStatisticHandler.java]: +

File: src/com/nkita/practice/collections/PersonStatisticHandler.java (hunk 2 above)
Comment (Ilya1222) [reply src/com/nkita/practice/collections/PersonStatisticHandler.java]: +

**Pull request merged**